worker/src/enclave/tls_ra.rs

```diff
@@ -27,19 +27,22 @@ extern "C" {
         retval: *mut sgx_status_t,
         socket_fd: c_int,
         sign_type: sgx_quote_sign_type_t,
+        skip_ra: c_int,
     ) -> sgx_status_t;
     fn request_key_provisioning(
         eid: sgx_enclave_id_t,
         retval: *mut sgx_status_t,
         socket_fd: c_int,
         sign_type: sgx_quote_sign_type_t,
+        skip_ra: c_int,
```

Comment:
I'll have to make sure to add this parameter in my PRs as well, since I moved the functions in the worker, and so the change will probably not get picked up by a simple rebase

Reply:
Yes, I think so too. I tried to be as little intrusive as possible here. :)